common/test/acceptance/pages/lms/teams.py

```diff
@@ -3,9 +3,20 @@
 Teams page.
 """
 
+from selenium.webdriver.common.keys import Keys
 from .course_page import CoursePage
 
 
+BROWSE_BUTTON_CSS = 'a.nav-item[data-index="1"]'
```

Comment:
I think it would be worth sharing some of this into a common paging class/mixin. As we add more paging UIs, they will all need these constants and some of these helper methods.